### PR TITLE
fix(web): album description

### LINKED ISSUE #6510: [BUG] New lines aren't added to descriptions
### The bug

When adding new lines in the description box for shared albums they appear in the editor but not on the page.

### The OS that Immich Server is running on

kubernetes / Ubuntu / Debian 

### Version of Immich Server

v1.93.1

### Version of Immich Mobile App

v1.92.0 build.117

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
Helm chart
```

### Your .env content

```Shell
Helm chart
```

### Reproduction steps

```bash
Attempt to add new lines to description of a shared album.
https://discord.com/channels/979116623879368755/994044917355663450/1197954639245615114
```

### PR DESCRIPTION
**Pull request by martabal**
## Changes made in this PR

- fixes #6510
- fixes sytle issues on the activity viewer

## Screenshots

| Before | After |
| :----: | :---: |
| Screenshot from 2024-01-19 20-25-30 | image  |

https://github.com/immich-app/immich/assets/74269598/4280b858-3e96-4fd9-8d5b-ec00e807d9d5

### CODE REVIEW

**Review by alextran1502 — APPROVED** [review]

File: web/src/routes/(user)/albums/[albumId]/+page.svelte [hunk 1]
```diff
@@ -142,6 +149,13 @@
     }
   });
 
+  const autoGrowHeight = () => {
+    // little hack so that the height of the text area is correctly initialized
+    textarea.scrollHeight;
+    textarea.style.height = '5px';
+    textarea.style.height = `${textarea.scrollHeight}px`;
+  };
```
Comment (jrasm91) [inline web/src/routes/(user)/albums/[albumId]/+page.svelte:157]: It looks like you just copied and pasted all of this stuff. Can you re-implement this without copy/pasting please?

Comment (martabal) [reply web/src/routes/(user)/albums/[albumId]/+page.svelte:157]: What do you want me to change ? 
Just put it in a `.ts` file ?

Comment (jrasm91) [reply web/src/routes/(user)/albums/[albumId]/+page.svelte:157]: Maybe look into a way to:
- add a directive/attribute/use:autogrow thing that will just do it automatically
- add a custom `TextArea` component that we can use everywhere we want a text box

also what is line 154?

Comment (martabal) [reply web/src/routes/(user)/albums/[albumId]/+page.svelte:157]: Without it, the textarea height is not correctly initialized:

https://github.com/immich-app/immich/assets/74269598/9ca34051-ddc9-4b5c-bc81-2f46b551a253

**Comment by ndragon798** [comment]
Hey @martabal it looks like this issue still happens on the album page if shared. 

Running 
image
Running latest 1.93.3

To recreate share the test album and open the share link.

**Comment by alextran1502** [comment]
@ndragon798 this PR is merged but hasn't been released yet. It will be available in the next release

**Comment by ndragon798** [comment]
Ah alright I thought it was included.
image
https://github.com/immich-app/immich/releases/tag/v1.93.3

**Pull request merged**